docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -121,6 +121,12 @@ const appList = [
     img: 'images/showcase/slimchess.png',
     link: 'https://slimchess.com',
   },
+  {
+    title: 'Serif.nu - Course planning for Northwestern University',
+    author: 'Joon Park',
+    img: 'images/showcase/serif-nu.png',
+    link: 'https://serif.nu',
+  },
```

Comment:
Feel free to add the `source` property for the github url `https://github.com/Joonpark13/serif.nu`.
As it's open source, you can move it right under the `Flow Dashboard` project 😄